java/model/src/main/java/com/instaclustr/cassandra/sidecar/model/result/OperationResult.java

```diff
@@ -0,0 +1,85 @@
+package com.instaclustr.cassandra.sidecar.model.result;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.UUID;
+
+import com.instaclustr.cassandra.sidecar.model.operation.Operation;
+import com.instaclustr.cassandra.sidecar.model.operation.OperationState;
+import com.instaclustr.cassandra.sidecar.model.operation.OperationType;
+
+public abstract class OperationResult extends Operation {
```

Comment:
Our code style is to usually avoid mutable "bean" style classes (private mutable fields, with getters and setters), and instead have `public final` fields which are assigned in a constructor.